test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -677,6 +677,8 @@ void UcxConnection::disconnect(UcxCallback *callback)
     assert(_establish_cb == NULL);
     assert(_disconnect_cb == NULL);
 
+    print_addresses();
```

Comment:
let's not print addresses on disconnect (IMO it's redundant)

Reply:
@yosefe Done